content/agent/proxy.md

```diff
@@ -144,56 +146,74 @@ Once the HAProxy configuration is in place, you can reload it or restart HAProxy
 
 **We recommend having a `cron` job that reloads HAProxy every 10 minutes** (usually doing something like `service haproxy reload`) to force a refresh of HAProxy's DNS cache, in case `app.datadoghq.com` fails over to another IP.
 
-Then edit each Agent to point to HAProxy by setting its `dd_url` to the address of HAProxy (e.g. haproxy.example.com). This `dd_url` setting can be found in `datadog.conf` for Agent v5 and `datadog.yaml` for Agent v6.
+### Datadog Agent configuration
+#### Agent v>6
+
+Edit each Agent to point to HAProxy by setting its `dd_url` to the address of HAProxy (e.g. `haproxy.example.com`). 
+This `dd_url` setting can be found in the `datadog.yaml` file. 
 
 `dd_url: https://haproxy.example.com:3834`
 
-If you want to send traces or processes through the proxy, you need to setup the following in `datadog.conf` for Agent v5:
+To send traces or processes through the proxy, setup the following in the `datadog.yaml` file:
 
 ```
-[trace.api]
-endpoint = https://haproxy.example.com:3835
+apm_config:
+    endpoint: https://haproxy.example.com:3835
 
-[process.api]
-url = https://haproxy.example.com:3836
+process_config:
+    url: https://haproxy.example.com:3836
 ```
 
-For Agent v6, set up the following in `datadog.yaml`:
+Then edit the `datadog.yaml` Agent configuration file and set `skip_ssl_validation` to `true`. This is needed to prevent Python from complaining about the discrepancy between the hostname on the SSL certificate (`app.datadoghq.com`) and your HAProxy hostname.:
```

Comment:
`prevent Python from complaining about the discrepancy [...]` is incorrect in the context of Agent v6: `make the Agent ignore the discrepancy [...]` would be better